### DISCUSSION TITLE
-alpha.#/-beta.# builds

### DISCUSSION THREAD

**Discussion by virtualex-itv** [MuhammedKalkan/OpenLens#112] [General]
If these are considered `"alpha"` or `"beta"` builds, shouldn't they be flagged as `"pre-release"`?

The reason behind the ask is that I am trying to keep the chocolatey package for this up-to-date and this is causing the auto-update feature to not detect these builds properly as "pre-release" so that they can be packaged and added to the chocolatey repository.

**Comment by MuhammedKalkan** [comment]
Lens repo does not tag them as prerelease. We are not fetching any of the tagged ones. And i dont think we should decide based on names. So there is that

**Reply by virtualex-itv** [reply]
Fair enough, I was able to figure out a workaround in this case and have pushed those versions to the chocolate repository although they will be flagged as prereleases automatically due to the word `alpha` or `beta` in the version name.  You may want to include that as an alternative method to install via chocolatey.

Chocolatey
`choco install -y openlens`

For alpha builds:
`choco install -y openlens --pre`

**Comment by MuhammedKalkan** [comment]
What about beta ones ?

**Reply by virtualex-itv** [reply]
~~I will add them as they were just released today...~~

added...
